bedrock/firefox/templates/firefox/faq.html

```diff
@@ -0,0 +1,105 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% from "macros-protocol.html" import hero with context %}
+
+{% block page_title %}{{ ftl('page-title') }}{% endblock %}
+{% block page_desc %} {{ ftl('page-desc') }} {% endblock %}
```

Comment:
nit: white space

This shows up in the meta description with whitespace at the beginning and end of the entry. 

```suggestion
{% block page_desc %}{{ ftl('page-desc') }}{% endblock %}
```